tests/ValueParsers/PhpDateTimeParserTest.php

```diff
@@ -81,143 +82,143 @@ public function validInputProvider() {
 		$valid = array(
 			// Normal/easy dates
 			'10/10/2010' =>
-				array( '+0000000000002010-10-10T00:00:00Z' ),
+				array( '+0000000000002010-10-10T00:00:00Z', 0, 0, 0, TimeValue::PRECISION_DAY, IsoTimestampParser::CALENDAR_GREGORIAN ),
```

Comment:
This reverts my patch https://gerrit.wikimedia.org/r/#/c/192300/ from February this year. :-( At least use
```php
$gregorian = 'http://www.wikidata.org/entity/Q1985727';
$julian = 'http://www.wikidata.org/entity/Q1985786';
```
as in `IsoTimestampParserTest` above.